ratpack-bearer-auth/src/main/java/st/ratpack/auth/ValidateTokenResult.java

```diff
@@ -1,6 +1,8 @@
 package st.ratpack.auth;
 
-public interface ValidateTokenResult {
+import java.io.Serializable;
+
+public interface ValidateTokenResult extends Serializable {
```

Comment:
Will making an interface serializable really be enough?

Reply:
Oauth token isn't serializable either

Reply:
It worked when I pulled it into Platform API running locally.

Reply:
Never mind. I was only testing invalid token results which wouldn't have an OAuthToken attached.